canonical/ubuntu.com
# Feature rebrand Landscape

## Done

- Rebranded Landscape bubble.
- Following pages have been rebranded:
  - landing page
  - /features
  - /managed
  - /pricing
  - /compare

## QA

- View the site in your web browser at https://ubuntu-com-14590.demos.haus/landscape
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check bubble throughout

## Issue / Card

Fixes #[WD-16365](https://warthogs.atlassian.net/browse/WD-16365)


[WD-16365]: https://warthogs.atlassian.net/browse/WD-16365?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
